lib/extensions/simple_form/components/html5.rb

```diff
@@ -0,0 +1,19 @@
+# Monkey-patch SimpleForm HTML5 module to remove redundant aria-required attribute
```

Comment:
I might wonder if we can override the behavior without monkeypatching the original by defining our own `html5_no_aria_required.rb` and swapping out `use :html5` with `use :html5_no_aria_required` from the code below? Might help avoid some of the risk with monkeypatching.

```rb
module SimpleForm
  module Components
    module HTML5NoAriaRequired
      include HTML5

      def html5_no_aria_required(_wrapper_options)
        input_html_options[:required] = input_html_required_option
        input_html_options[:'aria-invalid']  = has_errors? || nil
        nil
      end
    end
  end
end
```

Reply:
Yes. I like that approach. Revised that module.